src/chef-mover/rebar.config

```diff
@@ -62,8 +62,8 @@
 
     %%% Oldies
     % {ej, ".*",
-    %  {git,"git://github.com/seth/ej.git",
-    %    "132a9a3c0662a2377eaf7ebee694a496a0957160"}},
+    %  {git,"git://github.com/chef/ej.git",
```

Comment:
We can probably just get rid of this section.

Reply:
Done.